drivers/spi/Kconfig.renesas_ra

```diff
@@ -0,0 +1,33 @@
+# Renesas RA Family
+
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SPI_RENESAS_RA
+	bool "Renesas RA SPI"
+	default y
+	depends on DT_HAS_RENESAS_RA_SPI_ENABLED
+	select USE_RA_FSP_SPI
```

Comment:
Should we add	`select PINCTRL`?

Reply:
Thank you for your feedback, we have added it, Could you please re-check again?